### PR TITLE
Issues and update docs with make commands

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

**Review by khamui — COMMENTED** [review]
cool, issues fixed! just the command does not work somehow:

File: README.md [hunk 1]
```diff
@@ -27,8 +27,10 @@ Run
 
 Run with new deps (node v18.3.0)
 - Delete all deps `rm -rf node_modules`
-- Install fresh `npm install`
-- Start with cleared cache `expo start -c`
+- Install fresh `make install`
+- Start app `make start`
+- Start with cleared cache `make start-nocache`
```
[the remark this answers is not in the document]
Comment (phillimorland) [reply README.md:32]: which command and what does it say?

[the remark this answers is not in the document]
Comment (khamui) [reply README.md:32]: `make start-nocache`, something like `--nocache` unknown flag

**Comment by khamui** [comment]
very nice!!

**Pull request merged**